docs/graphql/manual/queries/query-filters.rst

```diff
@@ -3,7 +3,7 @@ Filter query results / search queries
 
 .. contents:: Table of contents
   :backlinks: none
-  :depth: 1
+  :depth: 2
```

Comment:
Changing this to 1 for this page. Its TOC is already too big. I think its time we break this page up into a few smaller pages.